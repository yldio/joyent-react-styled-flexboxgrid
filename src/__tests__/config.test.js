import config, { BASE_CONF, DIMENSION_NAMES } from '../config';

it('exports config', () => {
  expect(config).toMatchSnapshot();
});

it('exports BASE_CONF', () => {
  expect(BASE_CONF).toMatchSnapshot();
});

it('exports DIMENSION_NAMES', () => {
  expect(DIMENSION_NAMES).toMatchSnapshot();
});
