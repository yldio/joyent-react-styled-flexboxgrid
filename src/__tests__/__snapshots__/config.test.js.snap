// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports BASE_CONF 1`] = `
Object {
  "breakpoints": Object {
    "lg": 75,
    "md": 64,
    "sm": 48,
    "xs": 0,
  },
  "container": Object {
    "lg": 76,
    "md": 61,
    "sm": 46,
  },
  "gridSize": 12,
  "gutterWidth": 1,
  "mediaQuery": "only screen",
  "outerMargin": 2,
}
`;

exports[`exports DIMENSION_NAMES 1`] = `
Array [
  "xs",
  "sm",
  "md",
  "lg",
]
`;

exports[`exports config 1`] = `[Function]`;
